scylla-server-go/services/node_service/node_service.go

```diff
@@ -0,0 +1,32 @@
+package node_service
+
+import (
+	"errors"
+	"log"
+	"scylla-server/prisma"
+	"scylla-server/prisma/db"
+)
+
+func GetAllNodes() []db.NodeModel {
+	client, ctx := prisma.PrismaClient()
+
+	data, err := client.Node.FindMany(db.Node.DataTypes.Every()).Exec(ctx)
+	log.Printf("Err %q\n", err.Error())
+
+	return data
+}
+
+func UpsertNode(nodeName string) *db.NodeModel {
+	client, ctx := prisma.PrismaClient()
+
+	node, err := client.Node.FindUnique(db.Node.Name.Equals(nodeName)).Exec(ctx)
```

Comment:
Is there no prisma.upsert statement?

Reply:
There is but that's not what was used in the JS one.